plugin/vt_ida/plugin_loader.py

```diff
@@ -391,6 +390,29 @@ def write_config(self):
       return False
     return True
 
+  @staticmethod
+  def __compare_versions(current, new):
+    current_ver = current.split('.', 1)
+    new_ver = new.split('.', 1)
+
+    if len(current_ver) == 2 and len(new_ver) == 2:
+
```

Comment:
Add this sub-function:

def _normalize(a,b):
      while len(a) > len(b):
        b = '0' + b
      while len(b) > len(a):
        a = '0' + a
    return a,b


then use it multiple times to do the normalization of each field.